src/context.ts

```diff
@@ -71,7 +73,8 @@ const DefaultContext: ContextValue = {
   // The default tenderly dashboard for Yearn
   simulation: { dashboardUrl: "https://dashboard.tenderly.co/yearn/yearn-web" },
   cache: { useCache: true, url: "https://cache.yearn.finance" },
-  disableAllowlist: true
+  disableAllowlist: true,
+  partnerId: ""
```

Comment:
Just wondering whether we should have this as a default value, if we do and it's not overwritten by the consumer then they'll use the partner deposit contract with a partner of "" right? wasting some gas

Reply:
It would not since '' will evaluate to false

Reply:
I would rather use `partnerId === undefined` when its not provided

Reply:
Unfortunately we still need this change because we use the `Required` type here - https://github.com/yearn/yearn-sdk/blob/master/src/context.ts#L86 so it can't be optional. 

Made a PR to add it back - https://github.com/yearn/yearn-sdk/pull/256